src/simple_server/simple_game.clj

```diff
@@ -1,23 +1,33 @@
 (ns simple-server.simple-game)
 
 
-(def game-in-progress (atom nil))
+(def games-in-progress (atom {}))
+(def guesses (atom {}))
```

Comment:
Do you think it's a good idea to separate the two pieces of info which constitute a "game" into separate atoms?  Why?  How else might this be done?